index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="#" method="POST">
+
```

Comment:
- [ ] For this milestone, you're supposed to use the Formspree service to gather user data. Please make sure to change the action of the form to actually send the user data. You can find how to do it [here](https://formspree.io/html).